win: port traces test

ChandKV opened the pull request:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/pmem/nvml/1167)
<!-- Reviewable:end -->

krzycz commented:
Reviewed 10 of 10 files at r1.
Review status: all files reviewed at latest revision, 3 unresolved discussions, some commit checks failed.

---

*[src/test/traces/TEST0.PS1, line 69 \[r1\]](https://reviewable.io:443/reviews/pmem/nvml/1167#-KRDcH1NXzBF6o08aHBB:-KRDcH1OHR0vg_7m__AY:b-nbgp7m) ([raw file](https://github.com/pmem/nvml/blob/2071f5cd960ba2114a66773e2839aed1fe65f577/src/test/traces/TEST0.PS1#L69)):*
> ```
> 	exit 1
> }
> mv -Force .\custom_file.log-* custom_file.log
> ```

If destination file is `custom_file$UNITTEST_NUM.log`, then we can simply use "check" instead of "perl ..\match ..."
("check" is looking for log files that include current test number)
Same for Linux.

---

*[src/test/traces/TEST1.PS1, line 65 \[r1\]](https://reviewable.io:443/reviews/pmem/nvml/1167#-KRDZzd3TtVZh4KapaAW:-KRDZzd3TtVZh4KapaAX:b-89q77x) ([raw file](https://github.com/pmem/nvml/blob/2071f5cd960ba2114a66773e2839aed1fe65f577/src/test/traces/TEST1.PS1#L65)):*
> ```
> #   2. as a side benefit this will leave the error file in ASCII, so you can
> #      avoid a conversion from UNICODE to ASCII
> expect_normal_exit "cmd /c $Env:EXE_DIR\traces$Env:EXESUFFIX 2```>redir_stderr$Env:UNITTEST_NUM.log"
> ```

Cool!

---

*[src/test/traces/TEST1.PS1, line 68 \[r1\]](https://reviewable.io:443/reviews/pmem/nvml/1167#-KRDbutmMos5f2w_-P1E:-KRDbutnt0rk-CKAnp0U:b-z4l2c1) ([raw file](https://github.com/pmem/nvml/blob/2071f5cd960ba2114a66773e2839aed1fe65f577/src/test/traces/TEST1.PS1#L68)):*
> ```
> 
> # check results
> perl ..\match redir_stderr$Env:UNITTEST_NUM.log.match
> ```

Actually, this could be replaced with "check" (as in all the other TESTx scripts). (Also on Linux.)
Probably this is a copy&paste from TEST0.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1167)*
<!-- Sent from Reviewable.io -->

ChandKV commented:
Review status: 2 of 18 files reviewed at latest revision, 3 unresolved discussions.

---

*[src/test/traces/TEST0.PS1, line 69 \[r1\]](https://reviewable.io:443/reviews/pmem/nvml/1167#-KRDcH1NXzBF6o08aHBB:-KRFh51JkgkKwQXMj0-Y:bt2vr1b) ([raw file](https://github.com/pmem/nvml/blob/2071f5cd960ba2114a66773e2839aed1fe65f577/src/test/traces/TEST0.PS1#L69)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary>
> If destination file is `custom_file$UNITTEST_NUM.log`, then we can simply use "check" instead of "perl ..\match ..."
> ("check" is looking for log files that include current test number)
> Same for Linux.
</details>
Fixed.

---

*[src/test/traces/TEST1.PS1, line 68 \[r1\]](https://reviewable.io:443/reviews/pmem/nvml/1167#-KRDbutmMos5f2w_-P1E:-KREsrEDhJnHBSu0H9e9:b7rdij2) ([raw file](https://github.com/pmem/nvml/blob/2071f5cd960ba2114a66773e2839aed1fe65f577/src/test/traces/TEST1.PS1#L68)):*
<details><summary><i>Previously, krzycz (Krzysztof Czurylo) wrote…</i></summary>
> Actually, this could be replaced with "check" (as in all the other TESTx scripts). (Also on Linux.)
> Probably this is a copy&paste from TEST0.
</details>
Yes I pick this from TEST0 and prefixed it with perl as it was not auto resolved. Fixed this now to use check.

---


*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1167)*
<!-- Sent from Reviewable.io -->

krzycz commented:
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

---

Reviewed 17 of 17 files at r2.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/nvml/1167#-:-KRFlKvTwAZ7Ws39m1_2:bnfp4nl)*
<!-- Sent from Reviewable.io -->
